pocketlint/formatcheck.py

```diff
@@ -460,6 +460,12 @@ def check_regex_line(self, line_no, line):
                     icon='info',
                     )
 
+def check_semantic_newline(self):
+        """Check for a sentence with no new line."""
+        if self.line.find(['. ', '? ', '! ']) != (['\n', '..']):
+            self.message(
```

Comment:
This checks for
``. ``
``? ``
``! ``

and excludes new lines and ``..`` (for the adnotations)

not sure if I did it correctly though so I think tests would be useful

Previously it was only checking for full stop and space ``. ``